pkg/maps/policymap/policymap.go

```diff
@@ -251,6 +252,14 @@ func (key *PolicyKey) PortProtoString() string {
 	}
 }
 
+func portMaskToLength(v uint32) uint16 {
```

Comment:
This function is short, but pretty dense, would be nice to have a unit test here. Not blocking though.

Reply:
Yes please, unit tests and documentation of what this function is doing.

Reply:
I changed this to a map. Hopefully the comment above the map explains well enough.